docs/source/explore/get-started/install.rst

```diff
@@ -47,76 +73,93 @@ Once you've downloaded the installer, you're ready to install Vircadia. The proc
-3. Drag Vircadia to the Applications folder.                                               
-4. Open **System Preferences > General**.                                                        
-5. Next to the warning indicating that Interface is blocked, click 'Open Anyway'.                
-6. Confirm that you want to open the application.                                                 
+2. Agree to the License Agreement.
+3. Drag Vircadia to the Applications folder.
+4. Try opening Interface.app. You will be prompted that macOS will not open the file because it is unsigned.
+5. Open **System Preferences > Security & Privacy**. On older macOS versions this might be in **System Preferences > General**.
+6. Next to the warning indicating that Interface is blocked, click 'Open Anyway'.
+7. Confirm that you want to open the application.
+8. Allow microphone access if prompted, in order to be able to talk inside the application.
 
 At this point, Interface will open and you will be able to log in and begin exploring the metaverse.
 
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Linux Install
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+1. Mark the downloaded AppImage file as executable. (for example, with ``chmod +x Vircadia-Alpha-2020.2.5-Asteria.AppImage``.)
+2. (Optional) If you want the AppImage to be integrated into your system, install `AppImageLauncher <https://github.com/TheAssassin/AppImageLauncher>`_.
+3. Execute AppImage.
+
+Currently the server is not part of the Linux Interface AppImage. To host a domain, please refer to :doc:`Host a Domain from a Local or Cloud Linux Server <../../host/server-setup/linux-server>`.
+
 ----------------------------
 Upgrade Vircadia
 ----------------------------
 
+Vircadia is always changing, as we work to improve performance and add features that will enhance your experience in the metaverse.
+At any time, you can download the latest release from our `website <https://vircadia.com/download-vircadia/#interface>`_.
 
-.. Vircadia is always changing, as we work to improve performance and add features that will enhance your experience in the metaverse. If a new version has been released, you will be prompted to upgrade your installation the next time you run Vircadia. At any time, you can also download the latest release from our `website <https://www.highfidelity.com/download>`_.
-
-.. You cannot upgrade if Sandbox or the Console is running in the background of your computer. Be sure to quit these applications before upgrading.
-
-.. For Windows, locate the Vircadia app in your system tray. Right-click on the icon and select 'Quit'. Alternatively, end the 'server-console' background process using the Task Manager.
-.. For OS X, locate the Vircadia icon in the menu bar. Click on the icon and select 'Quit'.
+You cannot upgrade Vircadia on Windows if you have Vircadia Interface or Server running on your computer. Be sure to quit this application before upgrading.
```

Comment:
```suggestion
You cannot upgrade Vircadia on Windows if you have Vircadia Interface or Server running on your computer. Be sure to quit these applications before upgrading.
```

Reply:
Good catch. For some reason, Github didn't mark that in the former request: 
![1Auswahl_745](https://user-images.githubusercontent.com/11144627/93292919-09ac1500-f7e7-11ea-81a3-ba1d6d499f18.png)